### PR TITLE
Add logx pkg

### PR DESCRIPTION
**Pull request by gonzaloserrano**
I've removed the link to SP's confluence log guidelines and put instead the Dave Cheney's one.

### CODE REVIEW

File: logx/options.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package logx
+
+import "io"
+
+// Option is the common type of functions that set options
+type Option func(*options)
```
Comment (alexcarol) [inline logx/options.go]: does it make sense to have a public type with a private type as a parameter? I understand why we don't want to make options public, but this looks a bit "smelly"

Comment (gonzaloserrano) [inline logx/options.go]: if you make it unexported, then golint will warn that the `With*` constructors return an unexported type and will fail.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,25 @@
 # BSK [![Build Status](https://travis-ci.org/socialpoint-labs/bsk.svg?branch=master)](https://travis-ci.org/socialpoint-labs/bsk)
 
+## Packages
+
+### Extensions
```
Comment (ronnylt) [inline README.md]: I would generalize them as "packages", explaining when they add new important features or just when they are extensions to the standard libraries features. What do you think?

Comment (gonzaloserrano) [inline README.md]: OK, I just copy-pasted from the original README.

Comment (gonzaloserrano) [inline README.md]: I've changed it a little, what do you think?

**Pull request merged**